share/locale/en.json

```diff
@@ -1,165 +1,165 @@
     "systray_menu_exit": "Quit",
-    "systray_download_started_title": "OnionShare Download Started",
-    "systray_download_started_message": "A user started downloading your files",
-    "systray_download_completed_title": "OnionShare Download Finished",
-    "systray_download_completed_message": "The user finished downloading your files",
-    "systray_download_canceled_title": "OnionShare Download Canceled",
-    "systray_download_canceled_message": "The user canceled the download",
+    "systray_download_started_title": "OnionSharing",
+    "systray_download_started_message": "Files underway",
+    "systray_download_completed_title": "OnionSharing complete",
+    "systray_download_completed_message": "The user has your files now",
+    "systray_download_canceled_title": "OnionSharing stopped",
+    "systray_download_canceled_message": "The user stopped reception",
     "help_local_only": "Do not attempt to use Tor: For development only",
-    "help_stay_open": "Keep onion service running after download has finished",
-    "help_shutdown_timeout": "Shut down the onion service after N seconds",
-    "help_stealth": "Create stealth onion service (advanced)",
-    "help_receive": "Receive files instead of sending them",
-    "help_debug": "Log application errors to stdout, and log web errors to disk",
-    "help_filename": "List of files or folders to share",
-    "help_config": "Path to a custom JSON config file (optional)",
-    "gui_drag_and_drop": "Drag and drop files and folders\nto start sharing",
+    "help_stay_open": "Keep onion service running after OnionShare is completed",
+    "help_shutdown_timeout": "Close onion service after a given amount of seconds",
+    "help_stealth": "Make onion service stealthy (advanced)",
+    "help_receive": "Set your onion to reception instead of OnionSharing",
+    "help_debug": "Log OnionShare errors to stdout, and log web errors to disk",
+    "help_filename": "List of files or folders to OnionShare",
+    "help_config": "Location of custom JSON config file (optional)",
+    "gui_drag_and_drop": "Drag and drop files and folders\nto start OnionSharing",
     "gui_add": "Add",
     "gui_delete": "Delete",
     "gui_choose_items": "Choose",
-    "gui_start_server": "Start Sharing",
-    "gui_stop_server": "Stop Sharing",
-    "gui_stop_server_shutdown_timeout": "Stop Sharing ({}s remaining)",
-    "gui_stop_server_shutdown_timeout_tooltip": "Share will expire automatically at {}",
-    "gui_copy_url": "Copy Address",
+    "gui_start_server": "Start OnionSharing",
+    "gui_stop_server": "Stop OnionSharing",
+    "gui_stop_server_shutdown_timeout": "Stop OnionSharing ({}s remaining)",
+    "gui_stop_server_shutdown_timeout_tooltip": "OnionShare expires at {}",
+    "gui_copy_url": "Copy OnionShare Address",
     "gui_copy_hidservauth": "Copy HidServAuth",
-    "gui_downloads": "Download History",
-    "gui_downloads_window_tooltip": "Show/hide downloads",
+    "gui_downloads": "File History",
```

Comment:
As mentioned elsewhere - we are introducing the 'Upload' feature so we need to deliberately keep a differentiation between 'Download' and 'Upload' (I appreciate that in French, it's the same word, fun times!)

Reply:
I messed up here. Similarly, I think asking an audience to distinguish between Upload and Download narrows the userbase down rapidly. Even if understood, it isn't very comfortably familiar.
"OnionShare receptions"?

Reply:
I know what you mean by 'reception'/'receptions' but it doesn't make sense in everyday English. Reception to a native speaker usually refers to either:

1) A place in the front of an office/hotel
2) A mood/vibe ('the reception to my speech was cold')
3) A wedding reception (e.g an event)

It's not used for 'receiving' things in the common sense ('receiving' is in fact the better word).

Distinguishing between Upload and Download is an important feature of OnionShare. But we could potentially just use 'Sending' and 'Receiving', 'Sent' / 'Received' instead of uploaded/downloaded. Also that would therefore have a distinguished term in languages like French where upload/download is actually the same word and only context differentiates them.

@micahflee, thoughts on using 'Sending'/'Receiving' everywhere instead of downloading/uploading ?

Reply:
"Transmissions" are similarly bad. Shipments, is still bad, or maybe "deliveries"?